mbs_results/weighted_adj_val_time_series.py

```diff
@@ -0,0 +1,69 @@
+import numpy as np
+import pandas as pd
+from utils import convert_column_to_datetime
+
+
+def get_weighted_adj_val_time_series(filepath: str) -> pd.DataFrame:
+    """
+    Time series of weighted adjusted values by classification, question number,
+    and cell number.
+
+    Parameters
+    ----------
+    filepath : str
+        filepath to the asap output.
```

Comment:
Could this docstring be improved to be generalised (I see before it was `winsorisation output`)